Tests/GRPCTests/ConnectionPool/GRPCChannelPoolTests.swift

```diff
@@ -107,7 +108,7 @@ final class GRPCChannelPoolTests: GRPCTestCase {
     self.startChannel(withTLS: tls) {
       // We'll allow any number of waiters since we immediately fire off a bunch of RPCs and don't
       // want to bounce off the limit as we wait for a connection to come up.
-      $0.connectionPool.maxWaitersPerEventLoop = .max
+      $0.connectionPool.maxWaitersPerEventLoop = 1000
```

Comment:
The comment above seems to be out of step with this change.

Reply:
Thanks, didn't actually need this change.